From bzolnier@gmail.com  Fri Nov  6 14:48:04 2009
From: Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>
Date: Mon, 2 Nov 2009 16:25:32 +0100
Subject: Staging: rt28x0: Add proper selection of WIRELESS_EXT and WEXT_PRIV
To: "Greg Kroah-Hartman" <gregkh@suse.de>
Cc: Larry Finger <Larry.Finger@lwfinger.net>, linux-kernel@vger.kernel.org, devel@driverdev.osuosl.org
Message-ID: <200911021625.32165.bzolnier@gmail.com>


After the incorporation of the patch entitled "wext: refactor", some
of the wireless drivers in drivers/staging fail to build because they
need to have CONFIG_WIRELESS_EXT and CONFIG_WEXT_PRIV defined.

[ patch description borrowed from the previous fix for wireless staging
  drivers ("staging: Add proper selection of WIRELESS_EXT and WEXT_PRIV")
  authored by Larry Finger ]

Cc: Larry Finger <Larry.Finger@lwfinger.net>
Signed-off-by: Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/rt2860/Kconfig |    2 ++
 drivers/staging/rt2870/Kconfig |    2 ++
 2 files changed, 4 insertions(+)

--- a/drivers/staging/rt2860/Kconfig
+++ b/drivers/staging/rt2860/Kconfig
@@ -1,6 +1,8 @@
 config RT2860
 	tristate "Ralink 2860/3090 wireless support"
 	depends on PCI && X86 && WLAN
+	select WIRELESS_EXT
+	select WEXT_PRIV
 	---help---
 	  This is an experimental driver for the Ralink 2860 and 3090
 	  wireless chips.
--- a/drivers/staging/rt2870/Kconfig
+++ b/drivers/staging/rt2870/Kconfig
@@ -1,5 +1,7 @@
 config RT2870
 	tristate "Ralink 2870/3070 wireless support"
 	depends on USB && X86 && WLAN
+	select WIRELESS_EXT
+	select WEXT_PRIV
 	---help---
 	  This is an experimental driver for the Ralink xx70 wireless chips.
